static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -99,13 +78,24 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
             <TimeSince date={feedbackItem.timestamp} />
           </span>
           <div style={{gridArea: 'message'}}>
-            <TextOverflow>{feedbackItem.message}</TextOverflow>
+            <TextOverflow>{feedbackItem.metadata.message}</TextOverflow>
           </div>
           <Flex style={{gridArea: 'icons'}} gap={space(1)} align="center">
             <Flex align="center" gap={space(0.5)}>
-              <ProjectAvatar project={project} size={12} /> {slug}
+              <ProjectAvatar
+                project={feedbackItem.project}
+                size={12}
+                title={feedbackItem.project.slug}
```

Comment:
what does the `title` prop do?? i couldn't figure it out

Reply:
ugh, looks like it's doing nothing. I saw it someplace else and thought that it was the tooltip title :(

Reply:
same, i thought it would be the project slug text or something. so sad

Reply:
ripped it out. we can do `hasTooltip tooltip="text"` and it'll showup. leaving that for a followup PR

Reply:
nice